18.01.2019 16:57:53 Checking for system tray...
18.01.2019 16:57:53 Starting pgAdmin4 server...
18.01.2019 16:57:53 Creating server object, port:50770, key:5b99824b-f1de-434e-886e-f400fd8a9641, logfile:C:/Users/Administrator/.pgAdmin4.2123502176.log
18.01.2019 16:57:53 Python Path: C:/Program Files/PostgreSQL/10/pgAdmin 4/venv/Lib/site-packages;C:/Program Files/PostgreSQL/10/pgAdmin 4/venv/DLLs;C:/Program Files/PostgreSQL/10/pgAdmin 4/venv/Lib
18.01.2019 16:57:53 Python Home: C:/Program Files/PostgreSQL/10/pgAdmin 4/venv
18.01.2019 16:57:53 Initializing Python...
18.01.2019 16:57:53 Python initialized.
18.01.2019 16:57:53 Adding new additional path elements
18.01.2019 16:57:53 Redirecting stderr...
18.01.2019 16:57:53 stderr redirected successfully.
18.01.2019 16:57:53 Initializing server...
18.01.2019 16:57:53 Webapp Path: C:/Program Files/PostgreSQL/10/pgAdmin 4/web/pgAdmin4.py
18.01.2019 16:57:53 Server initialized.
18.01.2019 16:57:53 Starting Server Thread...
18.01.2019 16:57:53 Open the application code and run it.
18.01.2019 16:57:53 Set the port number, key and force SERVER_MODE off
18.01.2019 16:57:53 PyRun_SimpleFile launching application server...
18.01.2019 16:57:54 Application Server URL: http://127.0.0.1:50770/?key=5b99824b-f1de-434e-886e-f400fd8a9641
18.01.2019 16:57:54 The server should be up, we'll attempt to connect and get a response. Ping the server
18.01.2019 16:57:58 Attempt to connect one more time in case of a long network timeout while looping
18.01.2019 16:57:58 Everything works fine, successfully started pgAdmin4.
